src/utils/components/PrintButton.vue

```diff
@@ -0,0 +1,47 @@
+<script setup>
+import { ref, toRefs } from 'vue'
+
+import ModalPrintWithBackdrop from '@/utils/components/ModalPrintWithBackdrop.vue'
+import { useTippyTooltip } from '@/utils/composables/useTippyTooltip'
+
+const props = defineProps({
+    content: {
+        type: HTMLDivElement,
+        default: 0,
+    },
+})
+const { content } = toRefs(props)
+
+const emits = defineEmits(['hideParentModal'])
+
+useTippyTooltip('#PrintButtoneButton [data-tippy-content]')
+
+const showModal = ref(false)
+
+function printContent() {
+    showModal.value = true
+}
+function onHideParentModal(hide) {
+    emits('hideParentModal', hide)
+}
+</script>
+
+<template>
+    <button
+        id="PrintButton"
```

Comment:
Using an id here is not a good idea, usually you should have unique ID in the whole DOM of the application, so you might have several print button at the same time which will break the id uniqueness rule. I suggest to create a class, e.g. `print-button` and use the class selector in `useTippyTooltip('.print-button[data-tippy-content]')` (you don't need to define the class in CSS only in the element `class`).